core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/Ellipse2afp.java

```diff
@@ -56,6 +56,11 @@
 		B extends Rectangle2afp<?, ?, IE, P, V, B>>
 		extends RectangularShape2afp<ST, IT, IE, P, V, B> {
 
+    final String ELLIPSE_WIDTH_POSITIVE_OR_ZERO = "Ellipse width must be positive or zero";
```

Comment:
Hi Stephane,
I get this error during CI of my PRs. So the reason is the above
configurations. Is that right
"Failed to execute goal
org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (default) on
project math: Failed during checkstyle execution: There are 5 errors
reported by Checkstyle 6.18 with org/arakhne/afc/checkstyle/checks.xml
ruleset. ->"


On Sun, May 29, 2016 at 12:31 AM, Stéphane Galland <notifications@github.com
> wrote:

> In
> core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/Ellipse2afp.java
> <https://github.com/gallandarakhneorg/afc/pull/52#discussion_r64994587>:
>
> > @@ -56,6 +56,11 @@
> >  		B extends Rectangle2afp<?, ?, IE, P, V, B>>
> >  		extends RectangularShape2afp<ST, IT, IE, P, V, B> {
> >
> > +    final String ELLIPSE_WIDTH_POSITIVE_OR_ZERO = "Ellipse width must be positive or zero";
>
> In order to share these messages in all the classes for subclasses of
> Shape2afp, Shape2ai, what do you think to create the
> org.arakhne.afc.math.geometry.AssertionMessages that contains all the
> message constants for all the shape types?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/gallandarakhneorg/afc/pull/52/files/c8d39b6497aeb251c828ba1a5d33b33234c7a750#r64994587>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AR09BKpx7J6uh9BAp0HwgcgmnGsfgYDrks5qGLRJgaJpZM4IpKH_>
> .
>